src/RoslynInsertionTool/RoslynInsertionTool/CoreXT.cs

```diff
@@ -41,21 +48,43 @@ public static async Task<CoreXT> Load(GitHttpClient gitClient, string commitId)
                 DefaultConfigPath,
                 download: true,
                 versionDescriptor: vsBranch);
+            using var legacyPropsStream = await gitClient.GetItemContentAsync(
```

Comment:
do we need to handle cases where the legacy props file doesn't exist (e.g. older branches of VS?

Reply:
Fixed. Thanks for catching this.